src/components/SingleTweet.tsx

```diff
@@ -96,17 +96,15 @@ const IconBg = styled.div<IIconBg>`
   }
 `;
 
-const SingleTweet: React.FC<Tweet> = ({
-  userlogo,
-  username,
-  userNickname,
-  tweetText,
-  tweetDate,
-  lelekaLink,
-  retweetCount,
-  tweetQuoteCount,
-  likeCount,
-}) => {
+const SingleTweet = ({
+  // _id,
+  author,
+  content,
+  createdAt,
+  // repliedTo,
+  // updatedAt,
```

Comment:
```suggestion
  author,
  content,
  createdAt,
  updatedAt,
```
clean code
use updatedAt on form